### DISCUSSION TITLE
Amazon Lightsail

### DISCUSSION THREAD

**Discussion by maxiplux** [aws/copilot-cli#3233] [Ideas]
Please, would it be possible to add support to the Amazon Lightsail service ?.
Thanks for this amazing pilot for our projects.

**Comment by iamhopaul123** [comment]
Hello @maxiplux unfortunately we don't have a plan for supporting Lightsail right now. However, by looking at Lightsail, I feel like it is very similar to [App Runner service](https://aws.amazon.com/apprunner/) which we support: https://aws.github.io/copilot-cli/docs/concepts/services/#request-driven-web-service. Could you take a look at it and let us know if App Runner helps your projects?

**Comment by maxiplux** [comment]
Hi @iamhopaul123 , thanks for your quick reply. I don't like app Runner services, because that is a little complex for some customers in my scope. I am looking for something to allow my customers from developing countries can use without a complex process to set up or complex elements to understand their invoice. That is the reason because I love lightsail, a couple of clicks, and everything is ready to deploy. if you want, I can support you to add lightsail to AWS copilot. 

Please, let me know how can I support this beautiful project.

have a nice day.

**Comment by iamhopaul123** [comment]
Hello @maxiplux. I've tried using Lightsail and understand how beautiful experience for deploying a containerized app within a few clicks. However, I wonder how Copilot could possibly help, because Copilot essentially helps customers to use ECS more easily (within a couple of commands). I wonder how Lightsail could fit into Copilot's world: in Copilot customers usually need to create an [application](https://aws.github.io/copilot-cli/docs/concepts/applications/) first and then [environment](https://aws.github.io/copilot-cli/docs/concepts/environments/) to deploy [services](https://aws.github.io/copilot-cli/docs/concepts/services/).

It sounds to me Lightsail has already done the same thing under the hood. Let me know your thought about it. Thank you!
